modules/extensions-soap-support/src/main/java/org/mule/runtime/module/extension/soap/internal/loader/SoapInvokeOperationDeclarer.java

```diff
@@ -117,7 +117,7 @@ private void declareMetadata(OperationDeclarer operation, ClassTypeLoader loader
                                                                                 InvokeOutputTypeResolver::new,
                                                                                 NullMetadataResolver::new);
     operation.withModelProperty(new MetadataResolverFactoryModelProperty(() -> factory));
-    operation.withModelProperty(new MetadataKeyIdModelProperty(loader.load(WebServiceTypeKey.class), KEYS_GROUP));
+    operation.withModelProperty(new MetadataKeyIdModelProperty(loader.load(WebServiceTypeKey.class), KEYS_GROUP, "SoapInvoke"));
```

Comment:
Is this hardcoded value needed?

Reply:
it should be a constant (will change), but is indeed needed. That's the name of the Metadata category for the resolver, and in this case there's no need for it to be anonymous.